sources/packages/web/src/store/modules/auth/getters.ts

```diff
@@ -2,7 +2,7 @@ import { GetterTree } from "vuex";
 import { AuthState, RootState } from "../../states";
 
 export const getters: GetterTree<AuthState, RootState> = {
-  isAuthenticated(state: AuthState): boolean {
-    return state.isAuthenticated;
-  }
+    isAuthenticated(state: AuthState): boolean {
+        return state.isAuthenticated as boolean;
```

Comment:
I believe that "state.isAuthenticated" is already as boolean. Is there a reason why this change was needed?

Reply:
Typescript errored out unless this was done.